bin/yarn

```diff
@@ -3,11 +3,9 @@
 
 APP_ROOT = File.expand_path('..', __dir__)
 Dir.chdir(APP_ROOT) do
-  begin
-    exec 'yarnpkg', *ARGV
-  rescue Errno::ENOENT
-    $stderr.puts 'Yarn executable was not detected in the system.'
-    $stderr.puts 'Download Yarn at https://yarnpkg.com/en/docs/install'
-    exit 1
-  end
+  exec 'yarnpkg', *ARGV
```

Comment:
This is a rubocop-related change, now that we're allowing Ruby 2.5+ syntax (including block-level `rescue`) in its checks.